src/System.Net.Sockets/src/System/Net/Sockets/NetEventSource.Sockets.cs

```diff
@@ -73,6 +74,15 @@ public static void NotLoggedFile(string filePath, Socket socket, SocketAsyncOper
             }
         }
 
+        [NonEvent]
+        public static void NotLoggedFileStream(FileStream fileStream, Socket socket, SocketAsyncOperation completedOperation)
+        {
+            if (IsEnabled)
+            {
+                Log.NotLoggedFile(fileStream.Name, GetHashCode(socket), completedOperation);
+            }
+        }
```

Comment:
Do we need this method?  Could we instead just call `NotLoggedFile(stream.Name, socket, completedOperation)` at the call site?

Reply:
I am thinking about it when coding. I will remove this code and make direct call.